### PR TITLE
Add swipebox@1.4.4 w/ git auto-update 

### LINKED ISSUE #6913: [Request] swipebox.min.css
Library name: Swipebox
Git repository url: https://github.com/brutaldesign/swipebox
npm package url(optional): 
License(s): http://www.pixeden.com/license
Official homepage: http://brutaldesign.github.io/swipebox/

### PR DESCRIPTION
**Pull request by sashberd**
close #6913, cc @brutaldesign
Pull request for issue: #6913 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/brutaldesign/swipebox/tree/master
 * Official website (optional, not the repository): http://brutaldesign.github.com/swipebox
 * NPM package url (optional): https://www.npmjs.com/package/swipebox
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 108
   - Count of stars: 1840
   - Count of forks: 594
 * NPM download stats (optional):
   - Downloads in the last day: 29
   - Downloads in the last week: 135
   - Downloads in the last month: 580

# Essential checklist
 * [ ] I'm the author of this library
   * [] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@sashberd Could you provide any ref to show the npm package is official? Thanks.

**Comment by sashberd** [comment]
@maruilian11 
Sure
image

**Comment by maruilian11** [comment]
@sashberd hmmm... The git repo is same can't prove it's official. If there is a npm package link on git repo page. Or if this can be confirmed with the author will be great. Thanks.

**Comment by sashberd** [comment]
@maruilian11 I do not understand what is the problem?
If you enter to the https://www.npmjs.com/package/swipebox, there are links to official page, that the same as on github and link to github. However I opend the question on authors page, but the library was not maintained more then 2 years by author, thus I think the answer will not arrive

**Comment by maruilian11** [comment]
@sashberd Yes, but we can't take a risk to use an unsure npm package. Let's turn to use git auto-update.

**Comment by sashberd** [comment]
@maruilian11 So, if I understood right, you would like to remove version that I added manually from npm?

**Comment by maruilian11** [comment]
@sashberd We can keep the files if we can confirm the npm package is official. Maybe contact the author for sure.

File: ajax/libs/swipebox/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "swipebox",
+  "version": "1.4.6",
+  "description": "A touchable jQuery lightbox",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/brutaldesign/swipebox.git"
+  },
+  "filename": "jquery.swipebox.min.js",
```
Comment (maruilian11) [inline ajax/libs/swipebox/package.json]: Should be `js/jquery.swipebox.min.js` if we set `src` as base path. Thanks.

File: ajax/libs/swipebox/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "author": "brutaldesign <hello@csag.co>",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brutaldesign/swipebox.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "+(css|js)/*swipebox*"
```
Comment (maruilian11) [inline ajax/libs/swipebox/package.json]: We can directly put `"img/*"` under this line because their base path are the same.

**Comment by sashberd** [comment]
@maruilian11 The PR was updated. Thanks for you feedback. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by PeterDaveHello** [comment]
Both @maruilian11 & @extend1994 are right about the upstream source 👍

**Pull request merged**